rules/sinks/third_parties/sdk/segment/java.yaml

```diff
@@ -4,10 +4,74 @@
 
 sinks:
 
-  - id: ThirdParties.SDK.Segment
-    name: Segment
+  - id: ThirdParties.SDK.Segment.Android
+    name: Segment Analytics Android
     domains:
       - "segment.com"
     patterns:
       - "(?i)(com[.]segment[.]analytics[.]android|com[.]segment[.]analytics[.]java|io[.]segment[.]analytics[.]android).*"
     tags:
+
+  - id: ThirdParties.SDK.Segment.Java
+    name: Segment Analytics Java
+    domains:
+      - "segment.com"
+    patterns:
+      - "(?i)(com[.]segment[.]analytics[.]java).*"
+    tags:
+
+  - id: ThirdParties.SDK.Segment.Javascript
+    name: Segment Analytics Javascript
+    domains:
+      - "segment.com"
+    patterns:
+      - "(?i)(com[.]segment[.]analytics[.]javascript).*"
```

Comment:
Is this a Java Package?? 

As this rule file is `java.yaml` only java rule's should be present here

Similar comment for below rule for `python, ruby` etc

Reply:
removed non java rules